ArtifactBindingTest fails on quarkus 3 branch with ClassNotFoundException

### Before reporting an issue

- [X] I have searched existing issues
- [X] I have reproduced the issue with the latest release

### Area

testsuite

### Describe the bug

`ArtifactBindingTest` fails on quarkus 3 with a `ClassNotFoundException`, causing group `T6` to hang:

```
Error: Exception in thread "Thread-3243" jakarta.xml.ws.WebServiceException: Provider com.sun.xml.ws.spi.ProviderImpl not found
	at jakarta.xml.ws.spi.FactoryFinder$1.createException(FactoryFinder.java:35)
	at jakarta.xml.ws.spi.FactoryFinder$1.createException(FactoryFinder.java:32)
	at jakarta.xml.ws.spi.ServiceLoaderUtil.newInstance(ServiceLoaderUtil.java:76)
	at jakarta.xml.ws.spi.FactoryFinder.find(FactoryFinder.java:110)
	at jakarta.xml.ws.spi.Provider.provider(Provider.java:64)
	at jakarta.xml.ws.Endpoint.create(Endpoint.java:131)
	at org.keycloak.testsuite.util.ArtifactResolutionService.run(ArtifactResolutionService.java:127)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.ClassNotFoundException: com.sun.xml.ws.spi.ProviderImpl
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	at jakarta.xml.ws.spi.ServiceLoaderUtil.nullSafeLoadClass(ServiceLoaderUtil.java:62)
	at jakarta.xml.ws.spi.ServiceLoaderUtil.safeLoadClass(ServiceLoaderUtil.java:97)
	at jakarta.xml.ws.spi.ServiceLoaderUtil.newInstance(ServiceLoaderUtil.java:73)
	... 5 more
```

Which is probably happening due to `com.sun.xml.ws` dependency missing in `saml-core-api`.

### Version

quarkus3 branch

### Expected behavior

Test should pass

### Actual behavior

Test fails with `ClassNotFoundException` and causes the testsuite group `T6` to hang.

### How to Reproduce?

Run the test using the quarkus3 branch.

### Anything else?

_No response_

--- a/pom.xml
+++ b/pom.xml
@@ -116,6 +116,7 @@
         <sun.istack.version>3.0.10</sun.istack.version>
         <sun.saaj.version>2.0.1</sun.saaj.version>
         <sun.xml.bind.version>2.3.3-b02</sun.xml.bind.version>
+        <sun.xml.ws.version>4.0.0</sun.xml.ws.version>
         <javax.annotation-api.version>1.3.2</javax.annotation-api.version>
         <sun.activation.version>1.2.2</sun.activation.version>
         <org.glassfish.jaxb.xsom.version>2.3.3-b02</org.glassfish.jaxb.xsom.version>
@@ -356,6 +357,29 @@
                 <groupId>com.sun.xml.bind.external</groupId>
                 <artifactId>rngom</artifactId>
                 <version>${sun.xml.bind.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.sun.xml.ws</groupId>
+                <artifactId>rt</artifactId>
+                <version>${sun.xml.ws.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>com.fasterxml.woodstox</groupId>
+                        <artifactId>woodstox-core</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.glassfish.jaxb</groupId>
+                        <artifactId>txw2</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.glassfish.ha</groupId>
+                        <artifactId>ha-api</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>com.sun.xml.messaging.saaj</groupId>
+                        <artifactId>saaj-impl</artifactId>
+                    </exclusion>
+                </exclusions>
             </dependency>
             <dependency>
                 <groupId>org.glassfish.jaxb</groupId>

--- a/saml-core-api/pom.xml
+++ b/saml-core-api/pom.xml
@@ -61,6 +61,10 @@
             <groupId>com.sun.xml.messaging.saaj</groupId>
             <artifactId>saaj-impl</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.sun.xml.ws</groupId>
+            <artifactId>rt</artifactId>
+        </dependency>
     </dependencies>
     <build>
         <resources>

